### ISSUE TITLE
Support the simple HTML styling tags

### ISSUE THREAD

**Issue by anas-elgarhy** [Toqozz/wired-notify#116]
Most notifications demons support the ability to style the  text inside the notification using simple HTML tags like the `<b>` to make the text **bold** and `<i>` to make the text *italic*, at so on

## The expected output
* The notification title and bold **Playing** word in the body, something like this:

![image](https://github.com/Toqozz/wired-notify/assets/44965145/e661fe56-3c9f-46ac-bf56-35f27aa585fd)
> The image from dunst demon

##  The actual output
* Just the notification title and the actual `<b>` tags as a normal text

![image](https://github.com/Toqozz/wired-notify/assets/44965145/c231f904-08b5-49ed-93ce-8824605e346f)

**Comment by Toqozz** [comment]
Yeah, so our current method regrettably only lets you use these in the configuration.  We actually escape characters like `<` and `&`, which the only reason they're visible for this notification instead of rendering as you expect.

We need to update our escape code to allow those codes through.
